src/scripts/admin/blocks/card/container/deprecated.js

```diff
@@ -1,3 +1,5 @@
+/* eslint react/no-unknown-property: "off" */
```

Comment:
Can I ask why this was necessary? Were you trying to use a custom property/attribute on an element?